src/cli.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/* eslint no-console:0 */
+
+import minimist from 'minimist';
+import os from 'os';
+import path from 'path';
+import sh from 'shelljs';
+import tsts from 'tsts';
+
+const argv = minimist(process.argv.slice(2));
+
+// Get a full list of spells.
+const spells = sh.ls(path.join(__dirname, 'spells'))
```

Comment:
we should have an `index.js` in each directory that would return an object/map with each spell